endPoint/h2HttpHostEndPoint.js

```diff
@@ -107,7 +117,7 @@ export default class H2HttpHostEndPoint extends SecureHttpHostEndPoint {
             );
             const result = await this._service.processAsync(cms, fileContents);
             if (routingDataStep) routingDataStep.complete();
-            const [code, headerList, body] = await result.getResultAsync(
+            let [code, headerList, body] = await result.getResultAsync(
```

Comment:
Why const replace with let?
```suggestion
            const [code, headerList, body] = await result.getResultAsync(
```